docs/arcaflow/workflows/flow-control.md

```diff
@@ -115,16 +115,20 @@ The builtin function [bindConstants()](expressions.md#functions) allows you to f
 ```yaml title="input-repeat-name.yaml"
 iterations:
   - loop_id: 1
-    name: Matt
+    name: mogo
+    ratio: 3.14
```

Comment:
I think we want to structure this example so that the workflow output will be the same before and after refactoring the parent workflow _without_ changing the subworkflow.

I think that's going to be hard if the original input looks like this.

So, I think we want to "cheat" and claim that the original input looks something like
```yaml
iterations:
  - loop_id: 1
    data:
      name: mogo
      ratio: 3.14
  - ...
```
and that, after the refactor, the input looks something like
```yaml
data:
  name: mogo
  ratio: 3.14
iterations:
  - loop_id: 1
  - ...
```
because, in both cases the subworkflow will expect to get
```yaml
input:
  root: SubRootObject__RepeatedValues
  objects:
    SubRootObject__RepeatedValues:
      id: SubRootObject__RepeatedValues
      properties:
        constant:
          type:
            type_id: ref
            id: RepeatedValues
        item:
          type:
            type_id: ref
            id: SubRootObject
    RepeatedValues:
      id: RepeatedValues
      properties:
        name:
          type:
            type_id: string
        ratio:
          type:
            type_id: float
    SubRootObject:
      id: SubRootObject
      properties:
        loop_id:
          type:
            type_id: integer
```

Reply:
i think i did this, but with `repeated_inputs`